tasks/config.js

```diff
@@ -42,7 +42,8 @@ const shared = {
     tls: '{}',
     console: '{}',
     'require-dir': '{}'
-  }
+  },
+  timeout: 60000
```

Comment:
This should not be namespaced under `webpack`, as it is not a webpack relevant config. Better would be `test = {timeout: 60000}`